dts/bindings/gpio/infineon,mtbhal-gpio.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2020-2021 ATL Electronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: Infineon CAT1 GPIO PORT node
```

Comment:
should this perhaps be squashed with the previous commit that introduces the GPIO driver?

Reply:
agreed

Reply:
squashed